admin/config.yml

```diff
@@ -4,17 +4,17 @@ backend:
   base_url: https://cg-a72b65a7-73f0-4859-a1d7-dc953cf6ade8.app.cloud.gov/preview/gsa/search-gov-website/2020-redesign/
   auth_endpoint: external/auth/github
   preview_context: federalist/build
-  branch: 2020-redesign
+  branch: afarooque/2020-redesign-netlifycms
```

Comment:
Could the mismatch between this and line 4 be why the subnav menus aren't working?

Reply:
This is just controlling the branch the Netlify CMS interface commits to